Extension/src/LanguageServer/configurations.ts

```diff
@@ -965,6 +965,29 @@ export class CppProperties {
                     this.lastCustomBrowseConfiguration.Value = undefined;
                 }
             }
+
+            /*
+             * Ensure all paths are absolute
+             */
+            if (configuration.macFrameworkPath) {
+                configuration.macFrameworkPath = configuration.macFrameworkPath.map((path: string) => this.resolvePath(path, os.platform() === "win32"));
```

Comment:
Can the value of `os.platform() === "win32"` be saved and then reused in all the calls of `resolvePath`?